external/emulation/emulation/zhao_carr.py

```diff
@@ -68,6 +82,49 @@ def squash_precpd(state, emulator, cloud_squash):
 def apply_condensation_liquid_phase(state, net_condensation):
     # from physcons.f
     lv = 2.5e6
+    return apply_condensation(state, net_condensation, lv=lv)
+
+
+@numba.njit
+def ice_water_number(temperature_celsius, cloud):
```

Comment:
This is the fraction of condensate that is ice? The function name confused me... I was expecting a number concentration.

Reply:
I guess it's more a flag than a fraction.

Reply:
I renamed it to ice_water_flag.